test/adapter/polymorphic_test.rb

```diff
@@ -0,0 +1,72 @@
+require 'test_helper'
+
+module ActiveModel
+  class Serializer
+    module Adapter
```

Comment:
I hope to work with @remear to do some of these in cucumber so they're easier to compare, in general.  I suggested one file since yhe differences are subtle '. I'd be for separate files if we were more consistent in test file organization 

B mobile phone

> On Jan 20, 2016, at 11:00 AM, João Moura <notifications@github.com> wrote:
> 
> In test/adapter/polymorphic_test.rb:
> 
> > @@ -0,0 +1,72 @@
> > +require 'test_helper'
> > +
> > +module ActiveModel
> > +  class Serializer
> > +    module Adapter
> IMHO we should split this into different files under the adapters namespace for that sake of organization
> 
> —
> Reply to this email directly or view it on GitHub.
>